internal/component/otelcol/exporter/splunkhec/splunkhec.go

```diff
@@ -0,0 +1,85 @@
+// package splunkhec provides an otel.exporter splunkhec component
+
```

Comment:
if you volunteer to maintain the component, you should put your @ here like in the datadog community component:
// Maintainers for the Grafana Alloy wrapper:
//	- @adlotsof

Reply:
Realistically, i will have to discuss at work (with @PatMis16) - I would love to have the time in my free time, but its just not the case at the moment. I will come back to you on the `volunteering to be maintainer` - question.